controllers/validators/schema.json

```diff
@@ -0,0 +1,102 @@
+{
```

Comment:
It will be probably easier to write the objects in schemata in the Javascript object format. (Not worrying about the quotes etc.) Also indentation and other linting will be enforced that way. This file needs some.